src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingService.kt

```diff
@@ -166,6 +170,102 @@ class VurderingService(
         tilbakestillEndretTidForKopierteVurderinger(kopiAvVurderinger, tidligereVurderinger)
     }
 
+    @Transactional
```

Comment:
Hadde vært fint med en eller annen form av logging at dette skjer for behandling X, fra Y, då disse verdiene ikke kan plukkes ut fra url-path fra access-loggen

Reply:
Fint at du husket på `@Transactional`

Reply:
fixd 👍 [c593a9d](https://github.com/navikt/familie-ef-sak/pull/1740/commits/c593a9dea7933755dfa27eccbbbd10521a4ac189)